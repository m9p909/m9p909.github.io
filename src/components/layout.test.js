import React from "react"
import renderer from "react-test-renderer"
import Layout from './layout'
describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Layout siteTitle="Default Starter"> </Layout>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})