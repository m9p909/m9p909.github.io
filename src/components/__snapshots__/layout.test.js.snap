// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
Array [
  <nav
    className="navbar navbar-expand-lg navbar-light bg-light"
  >
    <div
      className="container"
    >
      <span
        className="navbar-brand"
      >
        Jack Clarke
      </span>
      <button
        aria-label="Toggle navigation"
        className="navbar-toggler collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse"
      >
        <div
          className="navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            className="nav-link"
            data-rr-ui-event-key={null}
            disabled={false}
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
          <a
            className="nav-link"
            data-rr-ui-event-key={null}
            disabled={false}
            href="/blog/"
            onClick={[Function]}
          >
            Blog
          </a>
          <a
            className="nav-link"
            data-rr-ui-event-key={null}
            disabled={false}
            href="/resume/"
            onClick={[Function]}
          >
            Resume
          </a>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="container"
  >
     
  </div>,
]
`;
